_sass/tools/_mixins.scss

```diff
@@ -0,0 +1,44 @@
+// gradient mixins
+@mixin c-hero-background {
+  width: 100%;
+  height: 100%;
+  background: url('../assets/logo-background.svg'), $top-gradient, url('../assets/homepage-clouds.jpg');
+  background-repeat: no-repeat;
+  background-size: 250%, cover, 200%;
+  background-position: center -10%, center center ,bottom center;
+  padding: 6rem 0;
+}
+
+// header font-styles
+@mixin main-heading-font {
+  font-family: $heading-font;
+  font-weight: 800;
+  font-style: normal;
+  font-size: 2rem;
+  line-height: 1.3;
+  margin: 0;
+  max-width: 299px;
+}
+
+// link styles
+@mixin default-link {
```

Comment:
default mixins are so great because they can be reusable and are really helpful when a project grows! nice job 💯

Reply:
💥